### PR TITLE
Kg maps take 2

### PR DESCRIPTION
**Pull request by katrinag**
@trang1618  re-uploading the mapping files - hopefully it works this time!

### CODE REVIEW

**Comment by trangdata** [comment]
Oh jeeze my bad I forgot to do `git fetch` before checking out the branch. 🤦🏽‍♀️  I'll work with this one. Do you want to close out the other PR? @katrinag Thank you! 🙏🏽

**Comment by katrinag** [comment]
I closed #14 so we can focus on this one now. :)

File: analyses/team3/reports/mapping.Rmd [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+Making a map like we did with the first few.  This time colors indicate how much the number of charges increased or decreased by between the periods. For most of the census tracts, it's close to zero.  
+
+```{r }
+changes_merged <- geo_join(tracts, changes, "GEOID", "tract_GEOID10")
+
+popup <- paste0("GEOID: ", changes_merged$GEOID, "<br>", "Number of charges", changes_merged$change)
+pal <- colorNumeric( # no bins this time, just did a gradient of colors
+  palette = "YlOrRd",
+  domain = changes_merged$change
+)
```
Comment (trangdata) [inline analyses/team3/reports/mapping.Rmd]: For dichotomous data like this, I suggest we use a divergent pallete, something like this:

```suggestion
# make vector of colors for values larger than 0 (6 colors)
rc1 <- colorRampPalette(colors = c("#3CBC75FF", "white"), space = "Lab")(6)
# make vector of colors for values larger than 0 (12 colors)
rc2 <- colorRampPalette(colors = c("white", "#440154FF"), space = "Lab")(12)
# combine color palettes
pal <- colorNumeric(palette = c(rc1, rc2), domain = changes_merged$change)
```

File: analyses/team3/reports/mapping.Rmd [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+- Making natural breaks with jenks (optimizes natural break points especially for map data): https://medium.com/@traffordDataLab/lets-make-a-map-in-r-7bd1d9366098
+
+- R color palletes: https://www.r-bloggers.com/2012/10/palettes-in-r/
+
+## Set Up
+
+I couldn't download a csv file from the website, so I had to copy and paste it into a notepad/excel file and save it as a csv.
+
+```{r }
+setwd("C:/Users/katri/Documents/GT/R-Ladies Datathon 2021")
```
Comment (trangdata) [inline analyses/team3/reports/mapping.Rmd]: If you double click at the `jat_datathon_3.Rproj` file, it should automatically open the Project and set directory to where we want. So I would remove this line.

**Review by trangdata — APPROVED** [review]
Looks awesome, @katrinag!!! 🥇 🌟 Thank you for all this work! I made a couple comments, but please feel free to merge whenever. My vote is showing only one map of the difference (for simplicity), but I'm open to other ideas! 📊

**Comment by katrinag** [comment]
Crap, I just saw that all the edits I made last night from the other branch aren't in this version....  I'll look into this later.  The code is essentially the same, but I had done some stuff to save files off as images.

**Comment by trangdata** [comment]
Oh no. Hope you find it. Could it be in the other branch we just closed?

File: analyses/team3/reports/mapping.Rmd (hunk 1 above)
Comment (katrinag) [reply analyses/team3/reports/mapping.Rmd]: Yes, I love this!  I just made this change in the most recent commit. :)

File: analyses/team3/reports/mapping.Rmd (hunk 2 above)
Comment (katrinag) [reply analyses/team3/reports/mapping.Rmd]: Updated!

**Comment by katrinag** [comment]
I think it was a saving/committing issue, but thankfully I didn't lose anything major.  I liked all your suggestions and I made the edits!  The changes map with the updated color scheme is in the mapping_files folder. :)

**Comment by trangdata** [comment]
Love it!!! Thank you!

**Pull request merged**